src/components/reusable/breadcrumbs/breadcrumbItem.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright Kyndryl, Inc. 2023
+ */
+
+// External library imports
+import { html, LitElement } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+// Relative component imports
+import styles from './breadcrumbItem.scss';
+
+/**
+ * Breadcrumb Item
+ *
+ * @slot unnamed - Slot for the content of the breadcrumb item, usually the label or text.
+ *
+ */
+@customElement('kyn-breadcrumb-item')
+export class BreadcrumbItem extends LitElement {
+  // Property to hold the URL for the breadcrumb item
+  @property({ type: String })
+  href = '';
```

Comment:
`href` property is not showing in storybook props table, may need to run analyze again.